### PR TITLE
added menu

### PR DESCRIPTION
**Pull request by jblss888**
It is in it's own method displayMenu()
-I also noticed i forgot to // out my print tests of loading the file so those are commented out now.

### CODE REVIEW

**Review by jcasta30 — DISMISSED** [review]
Tested it out and looks nice!

File: CustomerOrderDataBase.java [hunk 1]
```diff
@@ -57,26 +57,27 @@ public void loadFile() throws FileNotFoundException {
...
-            System.out.println(titles);
+            //Test System.out.println(titles);
 
             //This loop will extrapolate the data from each line and create the entryItem with the line's data
             while (in.hasNextLine()) {
                 String[] parsedLine = in.nextLine().split(",");
                 String date = parsedLine[0];
                 String cust_email = parsedLine[1];
-                String cust_location =parsedLine[2];
+                int cust_location = Integer.parseInt(parsedLine[2]);
```
Comment (daedren30) [inline CustomerOrderDataBase.java:67]:
```suggestion
                String cust_location = parsedLine[2]
```

**Review by daedren30 — APPROVED** [review]
Fixing OrderItem change (guessing it's from an earlier file)

File: CustomerOrderDataBase.java [hunk 2]
```diff
@@ -57,26 +57,27 @@ public void loadFile() throws FileNotFoundException {
...
-            System.out.println(titles);
+            //Test System.out.println(titles);
 
             //This loop will extrapolate the data from each line and create the entryItem with the line's data
             while (in.hasNextLine()) {
                 String[] parsedLine = in.nextLine().split(",");
                 String date = parsedLine[0];
                 String cust_email = parsedLine[1];
-                String cust_location =parsedLine[2];
+                String cust_location = parsedLine[2]
```
Comment (daedren30) [inline CustomerOrderDataBase.java:67]:
```suggestion
                String cust_location = parsedLine[2];
```

**Review by daedren30 — APPROVED** [review]
I forgot semicolon

**Review by daedren30 — APPROVED** [review]
Had to fix conflict with main branch because of viewOrder merge. Hope it's fine now.

**Review by daedren30 — APPROVED** [review]

**Review by Naomimmm — APPROVED** [review]

**Pull request merged**